onnx/common/ir.h

```diff
@@ -770,6 +770,58 @@ struct Node : public Attributes<Node> {
   }
 };
 
+// A class with the same properties as OperatorSetIdProto, but without protobuf
+// overhead, resulting in a simpler and more readable workflow.
+class OpSetID final {
+  private:
+    std::string domain_;
+    int64_t version_;
+
+  public:
+    explicit OpSetID(const OperatorSetIdProto& proto)
+      :domain_(proto.domain()), version_(proto.version()) {}
+
+    // Default Domain Constructor
+    explicit OpSetID(const int64_t version)
+      :domain_(""), version_(version) {}
+
+    explicit OpSetID(const std::string& domain, int64_t version)
+      :domain_(std::move(domain)), version_(version) {}
```

Comment:
Remove the std::move please. It's a const string reference, we are not suppose to change its content.